cli/command/environment.go

```diff
@@ -269,21 +288,31 @@ func (e *EnvironmentCommand) unlink(c *cli.Context) error {
 	}
 
 	if sourceEnvironmentID == destEnvironmentID {
-		return fmt.Errorf("Cannot unlink an environment from itself")
+		return fmt.Errorf("Cannot link an environment to itself")
```

Comment:
Shouldn't this be `link/unlink ...to/from` since this can happen in both scenarios?

Reply:
yup ✅